### DISCUSSION TITLE
Default filter

### DISCUSSION THREAD

**Discussion by grannymaster** [seerr-team/seerr#505] [Q&A]
Hey,

Is there a way to set a default filter for movies and series.  I usually set release date from this years January and onward, this way I get only the "new stuff". 

I'm wondering is there a way to set Jellyseer to default filter X?

**Comment by fallenbagel** [comment]
In the movies and series page?

**Reply by grannymaster** [reply]
Yes.

For example I want this to be always enabled.
![image](https://github.com/Fallenbagel/jellyseerr/assets/79359055/1c2894f7-f344-4f4c-aa9f-e4ddf3c4b212)

**Comment by jsmdy** [comment]
same question

**Comment by fallenbagel** [comment]
You can only set default filters on some stuff already (like language and region in general settings). As for other filters, there already is a feature request in github issues and you can follow that issue
